[BPK-2605] Icon only buttons always apply circular radius regardless of theming (#160)
Skyscanner/backpack-react-native: UNRELEASED.md

> Place your changes below this line.

**Fixed:**
- react-native-bpk-component-button

>   - `iconOnly` buttons no longer apply `boarderRadius` from theming. The iconOnly buttons now remain circular regardless of radius.

Reviewer: I think this can be more elegantly worded for consumers, how about:

```suggestion
  - `iconOnly` buttons no longer apply `borderRadius` from theming and are always circular.
```

>   - `iconOnly` buttons no longer apply `boarderRadius` from theming. The iconOnly buttons now remain circular regardless of radius.

Reviewer 2: ```suggestion
  - `iconOnly` buttons no longer apply `boarderRadius` from theming. `iconOnly` buttons now remain circular regardless of theming radius.
```
